packages/lit-dev-content/site/learn/learn.11tydata.js

```diff
@@ -4,187 +4,59 @@
  * SPDX-License-Identifier: BSD-3-Clause
  */
 
-const fs = require('fs').promises;
 const path = require('path');
-const matter = require('gray-matter');
 const { existsSync } = require('fs');
 
 const rootSiteDir = path.resolve(
   __dirname,
   '..',
 );
 
-const loadTutorialData = async (dirname) => {
-  const tutorialDir = path.resolve(
-    __dirname,
-    '../../samples/tutorials',
-    dirname
-  );
-  const encoding = {encoding: 'utf8'};
+const cleanUpTutorialData = (tutorials) => {
```

Comment:
I think the `wc-to-lit` tutorial needs to be filtered out since I believe it's a draft tutorial?